src/OrchardCore.Modules/OrchardCore.ContentFields/Views/HtmlField.Edit.cshtml

```diff
@@ -13,3 +13,21 @@
         <span class="hint">@settings.Hint</span>
     }
 </div>
+
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-mode-css" at="Foot"></script>
+<script asp-name="codemirror-mode-htmlmixed" at="Foot"></script>
+<script asp-name="codemirror-mode-javascript" at="Foot"></script>
+<script asp-name="codemirror-mode-xml" at="Foot"></script>
+<script at="Foot">
+    $(function () {
+        var optionsTexArea = document.getElementById('@Html.IdFor(x => x.Html)');
+        var editor = CodeMirror.fromTextArea(optionsTexArea, {
+            lineNumbers: false,
+            lineWrapping: true,
+            matchBrackets: true,
+            mode: { name: "htmlmixed" }
+        });
+        setTimeout(function() { editor.refresh(); },1);
```

Comment:
Why we need to refresh the editor?

Reply:
Without this, the style is only applied the first time the textarea has the focus.

Reply:
But we don't have it anywhere else right? Like for liquid editors. What's the difference?